pkg/tso/tso.go

```diff
@@ -230,6 +228,30 @@ func (t *timestampOracle) SyncTimestamp(leadership *election.Leadership) error {
 	return nil
 }
 
+func (t *timestampOracle) switchToNewPath(keyValue *mvccpb.KeyValue) error {
```

Comment:
As my previous comment mentioned, need configuration and switchToOldPath too to support bi-directional smooth switching.